Atomic/ps.py

```diff
@@ -83,6 +96,17 @@ def ps_tty(self):
         if not all_containers:
             return
 
+        if self.args.fstrim:
+            for info in all_containers:
+                container = info["container"]
+                ret = self._inspect_container(name=container)
```

Comment:
_inspect_container works only for Docker containers.  So this loop should skip any container where `info["type"] != "docker"`